### ISSUE TITLE
display a placeholder item to indicate the destination when the "Add SV/Container" modal is open

### ISSUE THREAD

**Issue by abey79** [rerun-io/rerun#4958]

**Comment by abey79** [comment]
@nikolausWest how we feel about something like this?


![Image](https://github.com/rerun-io/rerun/assets/49431240/4a874f2e-d981-47c9-84be-773321a7a125)

**Comment by nikolausWest** [comment]
The indentation in that example seems off by one (would expect it to look like it's a sibling node to "/" here, not a child) but other than that this seems reasonable to me (design is shippable but definitely room for improvement in the future)

**Comment by abey79** [comment]
Yeah, I agree the box should encompass the collapsing triangle area.
